### PR TITLE
WIP: updated branches for electron ID

### PR DESCRIPTION
**Pull request by bainbrid**
@jay-odedra please validate these changes, as discussed. Once ok'ed, I'll merge.
@gkaratha, FYI.

### CODE REVIEW

**Comment by bainbrid** [comment]
@nzipper

File: BParkingNano/plugins/ElectronMerger.cc [hunk 1]
```diff
@@ -189,15 +210,28 @@ void ElectronMerger::produce(edm::StreamID, edm::Event &evt, edm::EventSetup con
...
    ele.addUserInt("isLowPt", 0);
-   ele.addUserFloat("ptBiased", 20.);
-   ele.addUserFloat("unBiased", 20.);
-   ele.addUserFloat("mvaId", 20.);
-   ele.addUserFloat("pfmvaId", pf_mva_id);
-   ele.addUserInt("LooseID", ref->electronID("mvaEleID-Fall17-noIso-V1-wpLoose"));
-   ele.addUserInt("MediumID", ref->electronID("mvaEleID-Fall17-noIso-V2-wp90"));
-   ele.addUserInt("TightID", ref->electronID("mvaEleID-Fall17-noIso-V2-wp80"));
+   // Custom IDs
+   ele.addUserFloat("LPEleSeed_Fall17PtBiasedV1RawValue", 20.); // was called "ptBiased"
```
Comment (jay-odedra) [inline BParkingNano/plugins/ElectronMerger.cc:220]: A few branches for MVAID are set to 20 by default, is there a reason to keep them in?

Comment (gkaratha) [reply BParkingNano/plugins/ElectronMerger.cc:220]: This is for low pT. When you have an electron recoed only by PF you put that. But in this case we dont have LP at all for now at least

**Comment by bainbrid** [comment]
For the record, @jay-odedra has validated the electron ID related variables. 

For the PF electrons, we now have:
- the MVA raw score from the EGamma Fall17 ID,
- the wpLoose, wp80, and wp90 booleans from EGamma.

Jay has "synthesised" the boolean flags from the raw score, based on the default EGamma python configuration.

The attached slides show that we can reproduce the boolean flags from the raw scores, with an agreement at the 99% level. Good enough I'd say. So I'm going to merge this PR. 

[Electron MVA-id validation-2.pdf](https://github.com/DiElectronX/BParkingNANO/files/10918736/Electron.MVA-id.validation-2.pdf)

File: BParkingNano/plugins/ElectronMerger.cc (hunk 1 above)
Comment (bainbrid) [reply BParkingNano/plugins/ElectronMerger.cc:220]: Yep, placeholders. Added now just so it's not forgotten later.

**Pull request merged**